Sources/Utility/StringMangling.swift

```diff
@@ -12,6 +12,11 @@
 /// of them are programming-related.
 extension String {
 
+    /// Returns a form of the string that is a valid bundle identifier
+    public func mangledToBundleIdentifier() -> String {
+        return String(self.mangledToC99ExtendedIdentifier().characters.map { $0 == "_" ? "-" : $0 })
```

Comment:
This doesn't look right to me, I don't think it uses the C99 mangling, I think it uses another mangling strategy? Not that it matters so so much...